action.yml

```diff
@@ -1,5 +1,5 @@
 name: "Zimperium zScan"
-description: "​Scan your iOS & Android mobile app binaries to identify security, privacy, and compliance-related vulnerabilities. Free Trial Available!"
+description: "​Scan your iOS & Android mobile apps to identify security, privacy, and compliance-related vulnerabilities. Free Trial Available!"
```

Comment:
Why make this more vague?  As a developer with the original wording, I understand that "oh, it's going to scan the compiled/built binary" vs assuming "Oh cool, it's going to scan my source code".

Reply:
We can change that back. I am was trying to change the ACTIONS DESCRIPTION in the marketplace for search optimization by changing this. But that did not work.  We can change that back.

Reply:
We will need a new release for the actions to have a new change/wording on the marketplace action.

Reply:
https://github.com/Zimperium/zScanMarketplace/releases

Reply:
Do i need to do anything here?